lib/autospec/simple_runner.rb

```diff
@@ -16,18 +16,17 @@ def run(specs)
         self.abort
       end
       # we use our custom rspec formatter
-      args = ["-r", "#{File.dirname(__FILE__)}/formatter.rb"]
+      args = [
+        "-r", "#{File.dirname(__FILE__)}/formatter.rb",
+        "-f", "Autospec::Formatter"
+      ]
 
       command = begin
-        if ENV["PARALLEL_SPEC"] &&
+        if ENV["PARALLEL_SPEC"] != '0' &&
```

Comment:
I prefer to have this opt in for a few weeks, so do this for now:

` if ENV["PARALLEL_SPEC"] == '1'`